modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/util/ObjectEntryVariablesUtil.java

```diff
@@ -52,6 +52,17 @@
  */
 public class ObjectEntryVariablesUtil {
 
+	public static Map<String, Object> getActionOldVariables(
+		DTOConverterRegistry dtoConverterRegistry,
+		ObjectDefinition objectDefinition, JSONObject payloadJSONObject,
+		SystemObjectDefinitionMetadataRegistry
+			systemObjectDefinitionMetadataRegistry) {
+
+		return _getVariables(
+			dtoConverterRegistry, objectDefinition, payloadJSONObject,
+			systemObjectDefinitionMetadataRegistry, true);
+	}
+
 	public static Map<String, Object> getActionVariables(
```

Comment:
@p-albuquerque why do we need a new method instead of reusing the `getActionVariables`?